doc/cluttealtex.tex

```diff
@@ -1,296 +1,1504 @@
+}
+\parindent0pt
+
+% ----- Commands -----
+\newcommand\eg{e.\,g.\xspace}
+\newcommand\tos{$\to$\xspace}
+\let\TeXold\TeX
 \newcommand\CluttealTeX{ClutTeal\TeX\xspace}
 \providecommand\BibTeX{\textsc{Bib}\TeX\xspace}
-\newcommand\texcmd[1]{\texttt{\textbackslash #1}}
-\newcommand\texenv[1]{\texttt{#1}}
-\newcommand\texpkg[1]{\texttt{#1}}
-\newcommand\metavar[1]{\textnormal{\textsf{#1}}}
+\renewcommand\TeX{\TeXold\xspace}
+
+\NewTCBListing{boxcode}{O{none}O{\scriptsize}}{%
+	listing options={style=tcblatex,language={#1},basicstyle=\ttfamily#2},%
+	listing only,%
+}
+
+\NewTotalTCBox{\iboxcmd}{v}{%
+	verbatim,colupper=CtpBase,colback=CtpText,colframe=black,left=0mm,right=0mm,%
+}{%
+	\rule[-2.5pt]{0pt}{.85em}\lstinline[basicstyle=\ttfamily\normalsize,language=none,keywordstyle=\color{CtpMauve}\bfseries]^#1^%
+}
 
-\title{\CluttealTeX manual\\(Version 0.8.0)}
-\author{Lukas Heindl\thanks{thanks to ARATA Mizuki for writing cluttex of which \CluttealTeX is a fork}}
-\date{2024-03-11}
+\title{\CluttealTeX manual\\(Version 0.8.0)} % TODO(release) update
+\author{%
+	Lukas Heindl\thanks{thanks to ARATA Mizuki for writing cluttex of which \CluttealTeX is a fork}\\
+	\url{oss.heindl+latex@protonmail.com} \\
+	\url{https://github.com/atticus-sullivan/cluttealtex}
+}
+\date{2024-03-11} % TODO(release) update
 
 \begin{document}
 \maketitle
 \tableofcontents
 
-\chapter{About \CluttealTeX}
+\section{About \CluttealTeX}
 \CluttealTeX is an automation tool for \LaTeX\ document processing.
 Basic features are,
 \begin{itemize}
-\item Does not clutter your working directory with ``extra'' files, like \texttt{.aux} or \texttt{.log}.
-\item If multiple runs are required to generate correct document, do so.
-\item Watch input files, and re-process documents if changes are detected\footnote{needs an external program if you are on a Unix system}.
-\item Run MakeIndex, \BibTeX, Biber, if requested.
-\item Produces a PDF, even if the engine (e.g.\ p\TeX) does not suport direct PDF generation.
-  If you want a DVI file, use \texttt{--output-format=dvi} option.
-\item No external dependencies, \CluttealTeX only uses texlua which should come with TeXLive
+	\item Does not clutter your working directory with \enquote{auxiliary} files, like \texttt{.aux} or \texttt{.log}.
+	\item If multiple runs are required to generate the correct final document (with all references resolved etc), do so.
+	\item Watch input files, and re-process documents if changes are detected\footnote{needs an external program if you are on a Unix system}.
+	\item Run MakeIndex, \BibTeX, Biber, and some other tools if requested and necessary.
+	\item Produces a PDF, even if the engine (e.g.\ p\TeX) does not support direct PDF generation.
+		If you want a DVI file, use the \texttt{--output-format=dvi} option.
+	\item No external dependencies, \CluttealTeX only uses \texttt{texlua} which comes with more recent TeXLive (to my knowledge this is since \texttt{TeXLive-2008} the case).
```

Comment:
(github discarded my comment, argh -> only short version)

- reference https://www.preining.info/blog/2019/04/tex-live-2019-released/ texlive2019 since before that it wasn't Lua5.3 (and I didn't text cluttealtex with Lua5.2)
- probably safe to assume people use >=texlive2019 since this is what is shipped with debian+ubuntu versions (except those which reached EOL, like `focal` or `buster` which ship(ped) older versions of texlive)